File: calculaPreco.java
public class Produto {
    String nome;
    double preco;
    double desconto;

    public Produto(String nome, double preco, double desconto) {
        this.nome = nome;
        this.preco = preco;
        this.desconto = desconto;
    }

    public double calcularPrecoFinal() {
        return preco - (preco * desconto / 100);
    }

    public static void main(String[] args) {
        // 1. Desconto alterado de 15 para 25
        Produto produto = new Produto("Fone de ouvido", 200, 25);
        
        // 2. Nova linha adicionada para imprimir o Preço inicial
        System.out.printf("Preço inicial de %s: R$ %.2f%n", produto.nome, produto.preco);
        
        // 3. Linha para imprimir o Preço final
        System.out.printf("Preço final de %s: R$ %.2f%n", produto.nome, produto.calcularPrecoFinal());
    }
}

File: calculaPreco.cs
using System;

class Produto
{
    public string Nome { get; set; }
    public double Preco { get; set; }
    public double Desconto { get; set; }

    public double CalcularPrecoFinal()
    {
        return Preco - (Preco * Desconto / 100);
    }
}

class Program
{
    static void Main()
    {
        Produto produto = new Produto
        {
            Nome = "Fone de ouvido",
            Preco = 200,
            Desconto = 25 // 1. Desconto alterado de 15 para 25
        };
        
        // 2. Nova linha adicionada para imprimir o Preço inicial
        Console.WriteLine($"Preço inicial de {produto.Nome}: R$ {produto.Preco:F2}");
        
        // 3. Linha original do Preço final
        Console.WriteLine($"Preço final de {produto.Nome}: R$ {produto.CalcularPrecoFinal():F2}");
    }
}
